README.md

```diff
@@ -86,6 +86,7 @@ undoable(reducer, {
   redoType: ActionTypes.REDO, // define a custom action type for this redo action
 
   initialState: undefined, // initial state (e.g. for loading)
+  initActions: ['@@redux/INIT', '@@INIT'] // history will reset upon init action
```

Comment:
maybe this should be called `initTypes` to be more consistent with the `undoType`/`redoType` config options